Calendar/Home/Scene/HomeViewController.swift

```diff
@@ -8,33 +8,37 @@
 import UIKit
 
 final class HomeViewController: UIViewController, HomeView {
-  
+
   // MARK: - IBOutlet
-  
+
   @IBOutlet private weak var loginButton: UIButton!
-  
+
   // MARK: - Public property
-  
+
   var presenter: HomePresenter!
-  
+
   // MARK: - LifeCycle methods
-  
+
   override func viewDidLoad() {
     super.viewDidLoad()
-    
+
   }
-  
+
   // MARK: - Button Actions
-  
+
   @IBAction private func loginButtonTapped(_ sender: UIButton) {
     presenter.loginButtonTapped()
   }
-  
+
   @IBAction private func calendarPermissionButtonTapped() {
     presenter.calendarPermissionButtonTapped()
   }
-  
+
   @IBAction private func reminderPermissionButtonTapped() {
     presenter.reminderPermissionButtonTapped()
   }
+
+  @IBAction private func filterSelectStartDate() {
+    print("something")
```

Comment:
This will be updated when the UIDatePicker is added. Its a placeholder for that.